Core/src/org/sleuthkit/autopsy/report/FileReportText.java

```diff
@@ -1,7 +1,7 @@
 /*
  * Autopsy Forensic Browser
  *
- * Copyright 2013 Basis Technology Corp.
+ * Copyright 2018 Basis Technology Corp.
```

Comment:
This copyright notice should change to "Copyright 2013-2018 Basis Technology Corp", preserving the original date.